src/index.html

```diff
@@ -5,8 +5,413 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <!-- linthtml-configure attr-req-value="false" -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet"
+    crossorigin
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <a class= "container" href="#">
+        <img
+        src="https://raw.githubusercontent.com/mmahfuzi/layout_moyo-header-en/master/src/images/logo.png"
+        alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#laptops & computers"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="cards-container">
+      <div class="cards-container__page page">
+        <div class="card-block">
+          <main data-qa="card" class="card">
```

Comment:
After the header tag you should have one main tag that wraps the main content of the page, for this example the whole catalog(cards), and wrap each card with the div. Even better if you use section tag after the main and wrap each card with article tag for semantic purposes.